packages/modals/src/elements/Body.tsx

```diff
@@ -19,3 +19,5 @@ export const Body = React.forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivEleme
     return <StyledBody ref={ref} {...getContentProps(props)} />;
   }
 );
+
+Body.displayName = 'Body';
```

Comment:
Could you clarify why some components, such as `Modal.tsx` do not require a `displayName`?

Reply:
Most `HOC`s [apply their own display names](https://reactjs.org/docs/higher-order-components.html#convention-wrap-the-display-name-for-easy-debugging) to improve debugging. This is the case for `withTheme`.

Eslint is also limited in how it can detect React components https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md#about-component-detection

Reply:
Just realized that `Modal` no longer uses the `withTheme` HOC. I'll look into this more deeply.

Reply:
`forwardRef` has some [unique display name rules](https://reactjs.org/docs/forwarding-refs.html#displaying-a-custom-name-in-devtools).

To help with dev tool identification I'm going to add named functions to ensure components are shown correctly

![image](https://user-images.githubusercontent.com/4030377/75061144-f75a3a80-5494-11ea-833f-1afdc8fabf96.png)

Reply:
It turns out `forwardRef` was being identified in most areas successfully. The only "missing" display names were in `Modal` and `Tooltip` where the component returns a `createPortal()` call. I've added display names to both of these components now.